### DISCUSSION TITLE
Texture.fromURL() no longer available

### DISCUSSION THREAD

**Discussion by Flippsor** [pixijs/pixijs#10467] [v8 General] [answered]
The function Texture.fromURL() is no longer available. Is there a new way to create Texture from an URL i.e. with Assets.load etc.?

**Comment by bigtimebuddy** [comment]
```js
const texture = await Assets.load(url);
```

**Comment by Flippsor** [comment]
Already tried this but didn't work unfortunately.

I get following warning:

> [Assets] blob:https://localhost:4200/cb6abca5-c6ad-4974-b95d-28f66af6fce7 could not be loaded as we don't know how to parse it, ensure the correct parser has been added

Do i need to specify the parser? 

cb6abca5-c6ad-4974-b95d-28f66af6fce7 is the name of the asset.

What i did not mention before:
The image is recevied as an ArrayBuffer, converted to a Blob, converted to an ObjectURL and then put into Texture.fromURL

```
    const objBlob = new Blob([arrayBuffer]);
    const objectUrl = URL.createObjectURL(objBlob);
    const texture = Texture.fromURL(objectUrl);
```

**Comment by Flippsor** [comment, marked as answer]
I just tried different parser options and this works:

`await Assets.load<Texture>({src: path, loadParser: 'loadTextures'});`

**Reply by bigtimebuddy** [reply]
Yup! That is the correct way to load a Blob. Assets will load a file based off the extension. In this case Blob URL doesn't have the extension, so we need to provide `loadParser` to let Assets know to treat the request as a texture resource.
https://pixiplayground.com/#/edit/z3TM8euS-ww1_luv9z5Ld
